ui/building-pages.md

```diff
@@ -144,7 +144,7 @@ See the [Component Reference](./components/) for details about the different lib
 
 ## Dynamically Configuring Components with Expressions
 
-Virtually everywhere - with the notable exception of chart pages - every time you need a config prop to be dynamically updated, you can use an expression to configure it.
+Virtually everywhere - with the notable exception of config properties of user-created widgets - every time you need a config prop to be dynamically updated, you can use an expression to configure it.
```

Comment:
As I understand it now, this isn't true:

You can insert user-created widgets from another parent component and have their properties configured with expressions:

```yaml
# Personal widget
uid: widget_4064e9fb1e
tags: []
props:
  parameters:
    - description: A text prop
      label: Prop 1
      name: prop1
      required: false
      type: TEXT
  parameterGroups: []
component: f7-card
config:
  style:
    display: flex
    justify-content: center
slots:
  default:
    - component: oh-link
      config:
        text: =props.prop1
        class:
          - margin
```
```yaml
# Inserting the personal widget and configuring prop1 with an expression
component: widget:widget_4064e9fb1e
config:
  prop1: =2+3
```
![image](https://github.com/openhab/openhab-docs/assets/2004147/1b81c0ed-a898-48d5-b64c-297ca39137ed)

Reply:
Thanks for the hint! 
In the widget editor, properties are not evaluated, https://github.com/openhab/openhab-webui/pull/2007 adds a note about that.